src/pages/ProductDetail/components/Detail/Detail.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './Detail.scss';
+import DetailCount from './DetailCount';
+import DetailHeader from './DetailHeader';
+import Navigate from './Navigate';
+import TotalCost from './TotalCost';
```

Comment:
```suggestion
import React, { useState } from 'react';
import DetailCount from './DetailCount';
import DetailHeader from './DetailHeader';
import Navigate from './Navigate';
import TotalCost from './TotalCost';
import './Detail.scss';
```
import 순서도 잘 맞춰주세요.

위스터디 리팩토링 가이드에 나와있습니다. 참고 해 보세요

Reply:
반영했습니다.